### ISSUE TITLE
Allow Emptying Trash

### ISSUE THREAD

**Issue by fcheslack** [zotero/web-library#38]
We should allow emptying the entire trash with a single operation from the web library.

 - get the full list of items in the trash (/items/trash?format=keys)
 - confirm the user wants to permanently delete X items (?)
 - perform the multiple DELETE requests to permanently delete the items (probably in series, rather than parallel)
 - report the progress of the operation as it goes

Parts of this could potentially be re-used in other contexts, such as reporting progress for multi-request operations (bulk edit?)

Emptying the trash is easily the most requested such option, but we could also potentially apply other operations to all items in a view (rather than just the N items currently displayed) such as moving to a collection or tagging, but that involves harder UI questions.

**Comment by tnajdek** [comment]
We should probably wait for zotero/dataserver#5

**Comment by dstillman** [comment]
I don't think we want to support this in the API. Someone could have 100K items in the trash, so it can't necessarily be done in a single request. So I think doing it in batches with a progress meter makes more sense.
